src/components/Borrow/SwitchBorrow/ActionBorrow.tsx

```diff
@@ -0,0 +1,378 @@
+import { ExchangeCollateralTokens, ExchangeSyntheticTokens } from '@selectors/solanaWallet'
+import { BorrowTable } from '../BorrowTable/BorrowTable'
+import { BorrowedPair } from '../WrappedBorrow/WrappedBorrow'
+
+interface AssetPriceData {
+  priceVal: BN
+  assetScale: number
+  symbol: string | null
+  maxAvailable: BN
+  balance: BN
+}
+interface IProp {
+  cRatio: number
+  interestRate: number
+  liquidationPriceTo: number
+  liquidationPriceFrom: number
+  collateralRatioTo: number
+  collateralRatioFrom: number
+  nameButton: string
+  onClickSubmitButton: () => void
+  pairs: BorrowedPair[]
+  minCRatio: number
+  changeCRatio: (nr: number) => void
+  sending: boolean
+}
+export const ActionBorrow: React.FC<IProp> = ({
+  cRatio,
+  interestRate,
+  liquidationPriceTo,
+  liquidationPriceFrom,
+  collateralRatioTo,
+  collateralRatioFrom,
+  nameButton,
+  onClickSubmitButton,
+  pairs,
+  minCRatio,
+  changeCRatio,
+  sending
+}) => {
+  const classes = useStyles()
+  const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null)
+  const [amountBorrow, setAmountBorrow] = React.useState('')
+  const [amountCollateral, setAmountCollateral] = React.useState('')
+  const onClickPopover = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget)
+    setOption(true)
+  }
+  const onClosePopover = () => {
+    setAnchorEl(null)
+    setOption(false)
+  }
+  const [openOption, setOption] = React.useState(false)
+  const [customCRatio, setCustomCRatio] = React.useState('')
+  const [pairIndex, setPairIndex] = React.useState<number | null>(pairs.length ? 0 : null)
+
+  const getAssetInAndFor = (pair: BorrowedPair | null): [AssetPriceData, AssetPriceData] => {
+    if (pair === null) {
+      return [
+        {
+          priceVal: new BN(0),
+          assetScale: 0,
+          symbol: null,
+          maxAvailable: new BN(0),
+          balance: new BN(0)
+        },
+        {
+          priceVal: new BN(0),
+          assetScale: 0,
+          symbol: null,
+          maxAvailable: new BN(0),
+          balance: new BN(0)
+        }
+      ]
+    }
+    const collateral: AssetPriceData = {
+      priceVal: pair.collateralData.price.val,
+      assetScale: pair.collateralData.reserveBalance.scale,
+      symbol: pair.collateralData.symbol,
+      maxAvailable: pair.maxBorrow.val,
+      balance: pair.collateralData.balance
+    }
+
+    const synthetic: AssetPriceData = {
+      priceVal: pair.syntheticData.price.val,
+      assetScale: pair.syntheticData.supply.scale,
+      symbol: pair.syntheticData.symbol,
+      maxAvailable: pair.syntheticData.maxSupply.val,
+      balance: pair.syntheticData.balance
+    }
+
+    return [collateral, synthetic]
+  }
+  const [tokenFrom, tokenTo] = getAssetInAndFor(pairIndex !== null ? pairs[pairIndex] : null)
+
+  const changeCustomCRatio = () => {
+    customCRatio ? changeCRatio(Number(customCRatio)) : changeCRatio(minCRatio)
```

Comment:
merge these two ternaries into one if...else

Reply:
also, ternary operators are used to assign value to some variables basing on some conditions and here you aren't doing any assignments